tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = ui-tests, flake8
+skipsdist = True
+
+[testenv:ui-tests]
+passenv = DISPLAY
+deps = -rfrontend/test/ui/requirements/requirements.txt
+commands =
+    pytest -v --driver=Firefox --firefox-path=firefox/firefox frontend/test/ui {posargs}
+
+[testenv:flake8]
+skip_install = true
+deps = -rfrontend/test/ui/requirements/flake8.txt
+commands = flake8 frontend/test/ui
```

Comment:
It's probably an idea to have one linting job for all Python code in the repository, rather than make this specific to the UI tests.